### PR TITLE
A few small updates to L2 Page requested by the OP team

### PR DESCRIPTION
**Pull request by Rodrigolvc**
## Description

Optimism bridge should direct to https://app.optimism.io/bridge
Optimism description should read "Optimism is a fast, simple, and secure EVM-equivalent Optimistic Rollup. It scales Ethereum's tech while also scaling its values through retroactive public goods funding."
Crypto.com has withdraws / deposits for Optimism
OKX has withdraws for Optimism

## Related Issue
N/A

### CODE REVIEW

File: src/intl/en/page-layer-2.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
   "scaling-layer-1-with-shard-chains": "Scaling layer 1 with shard chains",
   "understanding-rollup-economics-from-first-principals": "Understanding rollup economics from first principals",
   "arbitrum-description": "Arbitrum is an Optimistic Rollup that aims to feel exactly like interacting with Ethereum, but with transactions costing a fraction of what they do on L1.",
-  "optimism-description": "Optimistic Ethereum is an EVM-equivalent Optimistic Rollup chain. It aims to be fast, simple, and secure.",
+  "optimism-description": "Optimism is a fast, simple, and secure EVM-equivalent Optimistic Rollup. It scales Ethereum's tech while also scaling its values through retroactive public goods funding.",
```
Comment (minimalsm) [inline src/intl/en/page-layer-2.json:125]: @corwintines was there a reason we wanted this shorter? just making sure this character length doesn't break any designs.


Also, we should stick to site conventions here... always "optimistic rollup" unless the former begins the sentence. I notice we're doing this everywhere... 😕 will fix in a separate PR.

```suggestion
  "optimism-description": "Optimism is a fast, simple, and secure EVM-equivalent optimistic rollup. It scales Ethereum's tech while also scaling its values through retroactive public goods funding.",
```

File: src/content/developers/docs/bridges/index.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ While there are many [types of bridge designs](https://blog.li.fi/what-are-block
 
 Bridges can usually be classified into one of the following buckets:
 
-- **Native bridges –** These bridges are typically built to bootstrap liquidity on a particular blockchain, making it easier for users to move funds to the ecosystem. For example, the [Arbitrum Bridge](https://bridge.arbitrum.io/) is built to make it convenient for users to bridge from Ethereum Mainnet to Arbitrum. Other such bridges include Polygon PoS Bridge, [Optimism Gateway](https://gateway.optimism.io/), etc.
+- **Native bridges –** These bridges are typically built to bootstrap liquidity on a particular blockchain, making it easier for users to move funds to the ecosystem. For example, the [Arbitrum Bridge](https://bridge.arbitrum.io/) is built to make it convenient for users to bridge from Ethereum Mainnet to Arbitrum. Other such bridges include Polygon PoS Bridge, [Optimism Gateway](https://app.optimism.io/bridge), etc.
```
Comment (minimalsm) [inline src/content/developers/docs/bridges/index.md:44]: I'll check if this link touches any other languages and fix in a separate PR.

File: src/data/layer-2/cex-layer-2-support.json [hunk 3]
```diff
@@ -7,13 +7,13 @@
...
-    "supports_deposits": ["Arbitrum"],
+    "supports_withdrawals": ["Arbitrum", "Optimism"],
+    "supports_deposits": ["Arbitrum", "Optimism"],
     "url": "https://crypto.com/"
   },
   {
     "name": "OKX",
-    "supports_withdrawals": ["Arbitrum"],
+    "supports_withdrawals": ["Arbitrum", "Optimism"],
     "supports_deposits": ["Arbitrum"],
```
Comment (minimalsm) [inline src/data/layer-2/cex-layer-2-support.json:17]: Nice 😎

File: src/intl/en/page-layer-2.json (hunk 1 above)
Comment (minimalsm) [reply src/intl/en/page-layer-2.json:125]: I'm wondering what the value of '_It scales Ethereum's tech while also scaling its values through retroactive public goods funding_' is to users. It's kinda vague and doesn't explain Optimism's unique selling point to those who don't already know about it... any thoughts @corwintines? No strong opinion on removing it, just don't think it achieves a lot here.

**Review by minimalsm — APPROVED** [review]
Thanks @rodrigolvc! Will get this in later today after Corwin gets the chance to see my question on the card description's line length but LGTM.

File: src/intl/en/page-layer-2.json (hunk 1 above)
Comment (corwintines) [reply src/intl/en/page-layer-2.json:125]: Pretty indifferent. It definitely adds an Optimism flair to the description which I can see as a reason for keeping it. No strong opinion on removing it though.

Comment (corwintines) [reply src/intl/en/page-layer-2.json]: Also, description length is fine. It was one of the shorter ones anyways. Now its comparable to Boba and Arbitrum.

**Review by corwintines — APPROVED** [review]
Thanks @Rodrigolvc!

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
@all-contributors please add @Rodrigolvc for docs, ideas

**Pull request merged**